### ISSUE TITLE
Android crashes after adding pcl nuget packages

### ISSUE THREAD

**Issue by wierzba3** [aritchie/userdialogs#325]
## Please check all of the platforms you are having the issue on (if platform is not listed, it is not supported)

 - [] iOS
 - [X] Android (works on API 19, does not work on 23)
 - [] UWP
 - [] PCL

## Version of ACR UserDialogs Library
6.3.7

## Expected Behaviour
App crashes after adding libraries. App works without them.

## Actual Behavior
_If applicable, please include entire exception - type, message, stacktrace_

Error message from device log:
```
01-28 00:50:17.157	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelUidCpuTimeReader		at libcore.io.IoBridge.open(IoBridge.java:452)
01-28 00:50:17.157	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelUidCpuTimeReader	java.io.FileNotFoundException: /proc/uid_cputime/show_uid_stat: open failed: ENOENT (No such file or directory)
01-28 00:50:17.157	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelUidCpuTimeReader		at com.android.server.ServiceThread.run(ServiceThread.java:46)
01-28 00:50:17.157	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelUidCpuTimeReader		at android.os.Looper.loop(Looper.java:148)
01-28 00:50:17.157	 5.7" Marshmallow (6.0.0) XHDPI Phone	Warning	487	KeyguardServiceDelegate	onScreenTurningOn(): no keyguard service!
01-28 00:50:17.157	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelUidCpuTimeReader		at java.io.FileInputStream.<init>(FileInputStream.java:103)
01-28 00:50:17.157	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelUidCpuTimeReader		at libcore.io.IoBridge.open(IoBridge.java:438)
01-28 00:50:17.157	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelUidCpuTimeReader		at java.io.FileInputStream.<init>(FileInputStream.java:76)
01-28 00:50:17.157	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelUidCpuTimeReader		at com.android.internal.os.KernelUidCpuTimeReader.readDelta(KernelUidCpuTimeReader.java:71)
01-28 00:50:17.157	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelUidCpuTimeReader		at com.android.internal.os.BatteryStatsImpl.updateCpuTimeLocked(BatteryStatsImpl.java:8031)
01-28 00:50:17.157	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelUidCpuTimeReader		at libcore.io.Posix.open(Native Method)
01-28 00:50:17.157	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelUidCpuTimeReader		at java.io.FileReader.<init>(FileReader.java:66)
01-28 00:50:17.157	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelUidCpuTimeReader		at libcore.io.BlockGuardOs.open(BlockGuardOs.java:186)
01-28 00:50:17.157	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelUidCpuTimeReader		at com.android.internal.os.BatteryStatsImpl$MyHandler.handleMessage(BatteryStatsImpl.java:155)
01-28 00:50:17.157	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelUidCpuTimeReader	Caused by: android.system.ErrnoException: open failed: ENOENT (No such file or directory)
01-28 00:50:17.157	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelUidCpuTimeReader		at android.os.HandlerThread.run(HandlerThread.java:61)
01-28 00:50:17.157	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelUidCpuTimeReader		... 10 more
01-28 00:50:17.157	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelUidCpuTimeReader	Failed to read uid_cputime
01-28 00:50:17.156	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelCpuSpeedReader		at java.io.FileInputStream.<init>(FileInputStream.java:76)
01-28 00:50:17.156	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelCpuSpeedReader		at libcore.io.IoBridge.open(IoBridge.java:438)
01-28 00:50:17.156	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelCpuSpeedReader	java.io.FileNotFoundException: /sys/devices/system/cpu/cpu0/cpufreq/stats/time_in_state: open failed: ENOENT (No such file or directory)
01-28 00:50:17.156	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelCpuSpeedReader		at libcore.io.IoBridge.open(IoBridge.java:452)
01-28 00:50:17.156	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelCpuSpeedReader	Failed to read cpu-freq
01-28 00:50:17.156	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelCpuSpeedReader		at android.os.Looper.loop(Looper.java:148)
01-28 00:50:17.156	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelCpuSpeedReader	Caused by: android.system.ErrnoException: open failed: ENOENT (No such file or directory)
01-28 00:50:17.156	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelCpuSpeedReader		at android.os.Handler.dispatchMessage(Handler.java:102)
01-28 00:50:17.156	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelCpuSpeedReader		at com.android.internal.os.BatteryStatsImpl.updateCpuTimeLocked(BatteryStatsImpl.java:8002)
01-28 00:50:17.156	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelCpuSpeedReader		... 10 more
01-28 00:50:17.156	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelCpuSpeedReader		at com.android.server.ServiceThread.run(ServiceThread.java:46)
01-28 00:50:17.156	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelCpuSpeedReader		at com.android.internal.os.BatteryStatsImpl$MyHandler.handleMessage(BatteryStatsImpl.java:155)
01-28 00:50:17.156	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelCpuSpeedReader		at libcore.io.BlockGuardOs.open(BlockGuardOs.java:186)
01-28 00:50:17.156	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelCpuSpeedReader		at android.os.HandlerThread.run(HandlerThread.java:61)
01-28 00:50:17.156	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelCpuSpeedReader		at com.android.internal.os.KernelCpuSpeedReader.readDelta(KernelCpuSpeedReader.java:49)
01-28 00:50:17.156	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelCpuSpeedReader		at libcore.io.Posix.open(Native Method)
01-28 00:50:17.156	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelCpuSpeedReader		at java.io.FileReader.<init>(FileReader.java:66)
01-28 00:50:17.156	 5.7" Marshmallow (6.0.0) XHDPI Phone	Error	487	KernelCpuSpeedReader		at java.io.FileInputStream.<init>(FileInputStream.java:103)
```

**Comment by aritchie** [comment]
Does this happen during execution of Userdialogs code or are you literally just installing the libraries?  I don't do any build weaving so installing the libraries on their own won't do anything 

Right now, there isn't anything in the message that says Acr.userdialogs and you haven't posted any code... You are asking me to play psychic at the moment

**Comment by wierzba3** [comment]
It happens after installing libraries and launching app ( crashes
immediately). I tried googling the error and I found some stuff related to
permissions required for a API 23+ does this sound familiar at all? It
might be another cause of the issue I suppose but the only change was
adding the library.

I will try to get more info.

On Jan 28, 2017 4:18 AM, "Allan Ritchie" <notifications@github.com> wrote:

Does this happen during execution of Userdialogs code or are you literally
just installing the libraries? I don't do any build weaving so installing
the libraries on their own won't do anything

Right now, there isn't anything in the message that says Acr.userdialogs
and you haven't posted any code... You are asking me to play psychic at the
moment

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/aritchie/userdialogs/issues/325#issuecomment-275842445>,
or mute the thread
